storage-model/src/main/java/no/unit/nva/publication/storage/model/DoiRequest.java

```diff
@@ -30,20 +31,24 @@
     private final Instant createdDate;
     @JsonProperty("customerId")
     private final URI customerId;
-    @JsonAlias("date")
+    @JsonProperty("owner")
     private final String owner;
+    @JsonProperty("resourceTitle")
```

Comment:
Duplicate the resource title in the doi request to facilitate the front-end view of the DoiRequests without joining the Resource with the DoiRequest